modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/portlet/action/ViewSharedAssetsMVCRenderCommand.java

```diff
@@ -36,15 +45,54 @@ public String render(
 
 		renderRequest.setAttribute(
 			ViewSharedAssetsDisplayContext.class.getName(),
-			_getViewSharedAssetsDisplayContextFactory.
-				getViewSharedAssetsDisplayContext(
-					renderRequest, renderResponse));
+			_getViewSharedAssetsDisplayContext(renderRequest, renderResponse));
 
 		return "/shared_assets/view.jsp";
 	}
 
+	private ViewSharedAssetsDisplayContext _getViewSharedAssetsDisplayContext(
```

Comment:
Hey @regiCesar21, move the repeated code to a Base class and apply to usages. Thx